Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/VisualThemes/ThemeEngines/InteractableRotationTheme.cs

```diff
@@ -46,6 +47,7 @@ public override void Init(GameObject host, ThemeDefinition settings)
             base.Init(host, settings);
 
             hostTransform = Host.transform;
+			originalRotation = hostTransform.localEulerAngles;
```

Comment:
the indentation on this seems off as well. Please update so that spacing is consistent.